### PR TITLE
Added Ceremony Honor Traits

### PR DESCRIPTION
**Pull request by Anonn12**
Added 4+ (average 5-6~) honor traits to each personality trait for ceremonies.

### CODE REVIEW

File: resources/dicts/events/ceremonies/ceremony_traits.json [hunk 1]
```diff
@@ -3,200 +3,432 @@
         "fearlessness",
         "valiance",
         "daring",
-        "bravery"
+        "bravery",
+        "adaptability",
+        "courage",
+        "valorousness",
```
Comment (scribblecrumb) [inline resources/dicts/events/ceremonies/ceremony_traits.json]: 'valour' might flow better than 'valourousness'

File: resources/dicts/events/ceremonies/ceremony_traits.json [hunk 2]
```diff
@@ -3,200 +3,432 @@
...
     "ambitious": [
         "initiative",
         "loyalty",
         "determination",
-        "bravery"
+        "bravery",
+        "passion",
+        "determination",
+        "commitment",
+        "desire"
```
Comment (scribblecrumb) [inline resources/dicts/events/ceremonies/ceremony_traits.json]: hmmm i'm not sure 'desire' really fits? perhaps 'drive' would work better.  'desire' just doesn't sound quite right.

File: resources/dicts/events/ceremonies/ceremony_traits.json [hunk 3]
```diff
@@ -3,200 +3,432 @@
...
     ],
     "bloodthirsty": [
         "ferocity",
         "strength",
         "skill",
-        "courage"
+        "courage",
+        "vigor",
+        "boldness",
+        "desire",
```
Comment (scribblecrumb) [inline resources/dicts/events/ceremonies/ceremony_traits.json]: same comment as the other 'desire'!

File: resources/dicts/events/ceremonies/ceremony_traits.json [hunk 4]
```diff
@@ -3,200 +3,432 @@
...
         "dedication",
         "forethought",
-        "strategy"
+        "strategy",
+        "adaptability",
+        "pragmatism",
+        "realism",
+        "observance",
+        "kindness",
+        "generosity"
```
Comment (scribblecrumb) [inline resources/dicts/events/ceremonies/ceremony_traits.json]: do 'kindness' and 'generosity' really fit here?

File: resources/dicts/events/ceremonies/ceremony_traits.json [hunk 5]
```diff
@@ -3,200 +3,432 @@
...
     ],
     "playful": [
         "enthusiasm",
         "spirit",
         "energy",
-        "determination"
+        "determination",
+        "humor",
+        "heart of gold",
+        "charism",
```
Comment (scribblecrumb) [inline resources/dicts/events/ceremonies/ceremony_traits.json]: i assume this is meant as 'charisma'? As 'charismatic' is already an actual trait, I'm not sure it should be an honor trait here.

File: resources/dicts/events/ceremonies/ceremony_traits.json [hunk 6]
```diff
@@ -3,200 +3,432 @@
...
         "boldness",
         "fearlessness",
         "valiance",
         "daring",
-        "bravery"
+        "bravery",
+        "vigor",
+        "selflessness",
+        "intrigue",
+        "remedial actions",
```
Comment (scribblecrumb) [inline resources/dicts/events/ceremonies/ceremony_traits.json]: maybe I'm just not used to the phrase, but 'remedial actions' sounds a little too formal to me

**Review by scribblecrumb — COMMENTED** [review]
Just a few notes!  You have done a good job tho, this really broadens the possibilities.

**Review by Ryos00 — APPROVED** [review]

**Pull request merged**